registry/template.test

```diff
@@ -0,0 +1,103 @@
+# This is a template for adding tests to the flutter/tests registry.
+#
+# To submit a new batch of tests, copy this file to create a new file
+# in this directory, update it as described below, and submit it as a
+# new PR to this repository.
+#
+# New files should be named `foo.test` where `foo` is a description of
+# the project whose tests are being added. (The file named
+# `template.test` is skipped automatically.)
+#
+# Lines beginning with a "#" are ignored. Delete all such lines from
+# this template before submitting a new `.test` file (you may add new
+# comments if you have something to say that doesn't fit in one of the
+# meaningful lines). Also delete blank lines like the following:
+
+# The contact lines must contain the e-mail addresses, one to a line,
+# of one or more people who will be able to help if a problem is found
+# with a test. People on this list should be able to respond to
+# e-mails within about 72 hours. If there is a problem with a test and
+# we cannot reach any of the people listed, then we will probably
+# remove the tests.
+contact=foo@example.com
+contact=bar@example.com
+
+# The fetch line(s) must contain one or more shell commands that fetch
+# the tests into a new directory called "tests" in the current
+# directory. The example below clones the dnfield/flutter_svg GitHub
+# repository then checks out a specific revision.
+#
+# Our current policy is that all tests should be in public GitHub
+# repositories. (This way, we don't take on additional dependencies on
+# other services that may or not go down. We already depend on GitHub,
+# so if GitHub goes down it doesn't matter that we can't reach
+# contributed tests.) If you have some tests to contribute that are
+# not and will not be in GitHub, please contact ian@hixie.ch before
+# submitting a PR proposing to add such tests. We may make exceptions
+# if your tests are especially valuable.
+#
+# The checkout must always check out the exact same set of tests. Do
+# not check out the current master branch or some such; always fetch a
+# specific revision. You may regularly submit PRs that update the
+# revision being used.
+#
+# Each command is run in a separate shell.
+#
+# The commands specified on the "fetch" lines must return the exit
+# code 0 or else the tests will be skipped entirely. The "git" program
+# will be on the path.
```

Comment:
Anything else they can rely being on the path?  Any forbidden executables?  What about package managers (yum, rpm) or curl (which would introduce a dependency on something other than GitHub even if it's not where their code is hosted)?

Reply:
\*shrug\*

We say the tests must be hermetic. I think it's up to the code reviewer to check that.